signature/mechanism_other.go

```diff
@@ -0,0 +1,13 @@
+//go:build !containers_image_openpgp && !containers_image_disable_signing && !cgo
+// +build !containers_image_openpgp,!containers_image_disable_signing,!cgo
+
+package signature
+
+// CgoIsDisabled indicates as a compiler error that this package requires cgo.
+//
+// The containers_image_openpgp build tag enables the OpenPGP implementation but
+// beware that it is not actively maintained and is considered insecure.
+//
+// The containers_image_disable_signing build tag disables signing and will
+// return an error when attempting to sign an image at runtime.
+var CgoIsDisabled = ContainersImageSignatureRequiresCgo
```

Comment:
Non-blocking: If we are in the business of grooming error messages, implementing `newGPGSigningMechanismInDirectory` and `newEphemeralGPGSigningMechanism` here, just to make this one error message stand out more, might be worthwhile.

Reply:
Will do.

Reply:
Okay, I've done that now. The error is now just:

```
./mechanism_other.go:16:21: undefined: ContainersImageSignatureRequiresCgo
```